Detectors/TRD/workflow/CMakeLists.txt

```diff
@@ -21,10 +21,11 @@ o2_add_library(TRDWorkflow
                        src/TRDTrackingWorkflow.cxx
                        src/EntropyDecoderSpec.cxx
                        src/EntropyEncoderSpec.cxx
+                       src/TrackBasedCalibSpec.cxx
                PUBLIC_LINK_LIBRARIES O2::Framework O2::DPLUtils O2::Steer O2::Algorithm O2::DataFormatsTRD O2::TRDSimulation O2::TRDReconstruction O2::DetectorsBase O2::SimulationDataFormat O2::TRDBase O2::TRDCalibration O2::GPUTracking O2::GlobalTrackingWorkflowReaders O2::GPUWorkflowHelper O2::ReconstructionDataFormats O2::TPCWorkflow O2::TRDWorkflowIO)
 
-                   #o2_target_root_dictionary(TRDWorkflow
-                   # HEADERS include/TRDWorkflow/TRDTrapSimulatorSpec.h)
+o2_target_root_dictionary(TRDWorkflow
```

Comment:
Found my own stupid mistake here. Don't know why I tried to create a root dictionary here. Probably because VdAndExBCalibSpec only has a header and therefore I did not add it to the sources above. I changed that now and that fixes the issue with the macro test